router/swagger.yaml

```diff
@@ -0,0 +1,22 @@
+swagger: "2.0"
```

Comment:
Legacy content?

Reply:
Connexion (and a lot of other tools) only supports swagger 2.0, so I'm using swagger here

Reply:
Sorry: I mean the file (there seems to be a more up-to-date copy in `/config`.

Reply:
This one's for the router, which s a separate service

Reply:
Aahh indeed!

Reply:
Ahh indeed it is!